### ISSUE TITLE
Not possible to disable a custom rule with inline comment

### ISSUE THREAD

**Issue by MiladSadinam** [linthtml/linthtml#489]
Hi,

I believe I have a found a bug which makes it impossible to disable a custom rule with an inline comment.

I have created a custom rule as a npm package which works fine. What I have:

```
module.exports = {
    name: 'dr-plano/i18n-not-splitted',
    lint(node, rule_config, { report }) {
        // TODO...
    }
};
```

The npm package is called `linthtml-plugin-i18n-not-splitted`. The package.json has the following content:
```
{
  "name": "linthtml-plugin-i18n-not-splitted",
}
```

I have added this to my `.linthtmlrc`:
```
{
  "plugins": [
    "linthtml-plugin-i18n-not-splitted"
   ],
  "rules": {
    "dr-plano/i18n-not-splitted": true,
  }
}
```
Until here, everything works fine. I was forced to use a namespace for the rule, as I was getting an error message when my custom rule name did not have a namespace.

Unfortunately, it is not possible now to disable the rule with an inline comment by using `<!-- linthtml-configure dr-plano/i18n-not-splitted="false" -->`. I had a look into the code, and it seems like the inline comment is always stripping away the namespace. But, the rules which are checked against are stored with a namespace.

Is this correct, or did I make a mistake somewhere?

Regards,
Milad

**Comment by KamiKillertO** [comment]
@MiladSadinam Thanks for reporting, I'll have a look to this 👍 .

>Until here, everything works fine. I was forced to use a namespace for the rule, as I was getting an error message when my custom rule name did not have a namespace.

This is normal, it's to avoid any naming conflicts between core rules and the plugin's rules, it's somehow documented [here](https://linthtml.vercel.app/developer-guide/plugins#expose-rules-in-plugins). I need to work on the documentation and improve it.

**Comment by KamiKillertO** [comment]
@MiladSadinam I have a fix ready. I'll release a new version soon.
By the way, if you only want to disable a rule you can use the `disable` inline instruction instead of the `configure` instruction.

```html
<p>text content checked</p>
<!-- linthtml-disable dr-plano/i18n-not-splitted -->
<p>text content not checked</p>
```

**Comment by MiladSadinam** [comment]
Thank you for the fix 🎉 And also for the hint.

**Comment by KamiKillertO** [comment]
@MiladSadinam I've just released a version with the fix 👍
